EasyPost.Tests/ServicesTests/WithParameters/ShipmentServiceTest.cs

```diff
@@ -332,7 +266,6 @@ public async Task TestGenerateForm()
 
             Parameters.Shipment.Buy buyParameters = new(rate)
```

Comment:
thought: if this is now empty, do we need to even supply blank params?

Reply:
It's not empty, it still included the  `id` of the `rate` to use when buying:
<img width="639" alt="image" src="https://github.com/EasyPost/easypost-csharp/assets/17054780/c15c469e-37c3-4e34-a6c8-1771f59e8ca0">